AGPs/AGP-W.md

```diff
@@ -0,0 +1,84 @@
+
+## Description of desired Association policy change
+
+The following changes are proposed to the `aragonpm.eth` aragonPM DAOs:
+
+- Mainnet and rinkeby environments:
+    - Change the root authority of the aragonPM DAOs from the 'Aragon One cold wallet' to the Aragon Association multisig. A Voting app within the DAO will control creating new repos and publishing new versions to all the repos (see Appendix for the proposed DAO structure)
+    - Create a new aragonPM registry at `a1.aragonpm.eth` controlled by Aragon One, granting 'Aragon One cold wallet' the `CREATE_REPO_ROLE` permission in the registry and setting this address as the root authority of the DAO. This registry can be used by Aragon One to publish apps before releasing them in `aragonpm.eth`.
+- Staging: Aragon One will keep control of the current staging environment, renaming it to `a1-staging` (in both https://aragonpm.com and deployments documentation). The rationale being that other teams can very easily create their own staging environments if they need one (the deployment process is documented in [aragon/deployments](https://github.com/aragon/deployments)) or use the `open.aragonpm.eth` registry that allows anyone to create their own repos.
+
+## Appendix: `aragonpm.eth` DAO structure
+
+### Current state
+
+- [Mainnet permissions](https://github.com/aragon/deployments/blob/master/environments/mainnet/permissions.yml)
+- [Rinkeby permissions](https://github.com/aragon/deployments/blob/master/environments/rinkeby/permissions.yml)
+
+### Proposed DAO structure
+
+#### New apps
+
+**Token Manager**:
+
+- Token: APM
+- Transferable: No
+- Maximum tokens per account: No limit
+
+**Voting**:
+
+- Token: APM
+- Support required: 66.666666666666666666% 
+- Minimum acceptance quorum: 50%
+- Vote duration: 72 hours
+
+#### Permissions
+
+|         App        |        Role        |      Grantee      |     Manager     |
+|:------------------:|:------------------:|:-----------------:|:---------------:|
+|       Kernel       |     MANAGE_APPS    |  AA multisig (*)  | AA multisig (*) |
+|         ACL        | CREATE_PERMISSIONS |    APMRegistry    | AA multisig (*) |
+|         ACL        | CREATE_PERMISSIONS |  AA multisig (*)  |   AA multisig   |
+|     APMRegistry    |    CREATE_REPOS    |     Voting (*)    | AA multisig (*) |
+| All Repo instances |   CREATE_VERSION   |     Voting (*)    | AA multisig (*) |
+|       Voting       |    CREATE_VOTES    | Token Manager (*) | AA multisig (*) |
+|       Voting       |   MODIFY_SUPPORT   |  AA multisig (*)  | AA multisig (*) |
+|       Voting       |    MODIFY_QUORUM   |  AA multisig (*)  | AA multisig (*) |
+|    Token Manager   |        MINT        |  AA multisig (*)  | AA multisig (*) |
+|    Token Manager   |        BURN        |  AA multisig (*)  | AA multisig (*) |
+
+- Mainnet AA multisig: 0xcafE1A77e84698c83CA8931F54A755176eF75f2C
+- Rinkeby AA multisig: TBD by Aragon Association
```

Comment:
From previous experience with the AGP-10 Community Funding DAO we may want to ensure that the address here is specified just so there is no misunderstanding later. Although the stakes are low as this is just a Rinkeby DAO, I thought it worth pointing out.

Reply:
Thanks for pointing it out, given that this is just for Rinkeby I would leave it as it is now.